web/src/views/InvestigationDetail.vue

```diff
@@ -486,8 +494,57 @@ export default defineComponent({
       clampBoundsX(false);
     }
 
+    const similarNuclei: Ref<Array<DetectedStructureWithColor>> = ref([]);
+    const axios = inject('axios') as AxiosInstance;
+    function colorSimilarNuclei(nuclei: Array<DetectedStructure>) {
+      const colors = ['#FF0000', '#00FF00', '#0000FF', '#FFFF00', '#FF00FF', '#00FFFF'];
```

Comment:
Don't use these colors. If you need categorical colors (which, see below), pull in D3's categorical colormaps, and use "category10".

Categorical colors to distinguish the markers may be the right way to go, but it could also be interesting to use a linear colormap to encode how similar each nucleus is. That might not be a good idea if you're always showing the 5 most similar (which presumably would all be close to the same color on a linear scale).